ebos/ecltransmissibility_impl.hh

```diff
@@ -1038,7 +1043,7 @@ applyNncMultreg_(const std::unordered_map<std::size_t,int>& cartesianToCompresse
     const auto& inputNNC = this->eclState_.getInputNNC();
     const auto& transMult = this->eclState_.getTransMult();
 
-    auto compressedIdx = [&cartesianToCompressed](const std::size_t globIdx)
+    auto compressedIdx = [&cartesianToCompressed, this](const std::size_t globIdx)
```

Comment:
"this" does not seem to be used. Please don't capture it.

Reply:
Thanks, it was a left over of an old-wrong change. I'll remove it